tests/test_compat.py

```diff
@@ -0,0 +1,40 @@
+#!/bin/python3
+
+# SPDX-FileCopyrightText: 2021 Jens Erdmann
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import os
+import sys
+import unittest
+
+TEST_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
As mentioned earlier already, I would like this sys.path manipulation to be move into a class function

```python
@classmethod
def setup_class(cls):
    sys.path.insert(0, os.path.abspath(os.path.dirname(__file__) + "/../"))

    try:
        from flict.flictlib import flict_config
        from flict.impl import FlictImpl
    except ImportError as e:
        self.fail("Can't import flict... {msg}".format(e))
```

Reply:
Should be fixed by now.